test/gen_rpc/multi_rpc_functional_SUITE.erl

```diff
@@ -0,0 +1,371 @@
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, exit, ['fatal']),
+    % Nothing blows up on sender side after sending call to the ether
+    true = erlang:is_process_alive(whereis(gen_rpc_server_sup)),
+    true = erlang:is_process_alive(whereis(gen_rpc_acceptor_sup)),
+    true = erlang:is_process_alive(whereis(gen_rpc_client_sup)).
+
+safe_eval_everywhere_mfa_throw_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_throw_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, throw, ['throwXup']),
+    ok = ct:pal("[erlang:throw only]. Verify the crash log from ct. You should see {{nocatch,throwXup}, ....} on the target node").
+
+safe_eval_everywhere_mfa_timeout_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [eval_everywhere_mfa_timeout_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, throw, ['throwXup']),
+    ok = ct:pal("erlang:throw only]. Verify the crash log from ct. You should see {{nocatch,throwXup}, ....} on the target node").
+
+client_inactivity_timeout(_Config) ->
+    ok = ct:pal("Testing [client_inactivity_timeout]"),
+    {_Mega, _Sec, _Micro} = gen_rpc:call(?SLAVE1, os, timestamp),
+    ok = timer:sleep(600),
+    %% Lookup the client named process, shouldn't be undefined. Rewrite/Remove test?
+    undefined =:= whereis(?SLAVE).
+
+server_inactivity_timeout(_Config) ->
+    ok = ct:pal("Testing [server_inactivity_timeout]"),
+    {_Mega, _Sec, _Micro} = gen_rpc:call(?SLAVE1, os, timestamp),
+    ok = timer:sleep(600),
+    %% Lookup the client named process, shouldn't be there
+    [] = supervisor:which_children(gen_rpc_acceptor_sup),
+    %% The server supervisor should have no children
+    [] = supervisor:which_children(gen_rpc_server_sup).
+
+%%% ===================================================
+%%% Auxiliary functions for test cases
+%%% ===================================================
+start_slaves() ->
+    ok = start_slave(?SLAVE_NAME1, ?SLAVE1),
+    ok = start_slave(?SLAVE_NAME2, ?SLAVE2),
+    ok = ct:pal("Slaves started"),
+    ok.
+
+start_slave(Name, Node) ->
+    %% Starting a slave node with Distributed Erlang
+    {ok, _Slave} = slave:start(?SLAVE_IP, Name, "+K true"),
+    ok = rpc:call(Node, code, add_pathsz, [code:get_path()]),
+    %% Start the application remotely
+    {ok, _SlaveApps} = rpc:call(Node, application, ensure_all_started, [gen_rpc]),
+    {module, gen_rpc_test_helper} = rpc:call(Node, code, ensure_loaded, [gen_rpc_test_helper]),
+    ok.
+
+stop_slaves() ->
+    Slaves = [?SLAVE1, ?SLAVE2],
+    [begin 
+        ok = slave:stop(Node)
+     end || Node <- Slaves],
+    ok = ct:pal("Slaves stopped").
+
+%% This is the middleman process listening for messages from slave nodes
+%% Then relay back to test case Pid for check.
+spawn_listener(_, _, _, Count) when Count =< 0-> ok;
```

Comment:
Only this test uses this. Do you like to put this int helper?
It may be useful for other tests later on?